src/config/__init__.py

```
@@ -1,3 +1,7 @@
-from src.config.settings import get_settings, get_database_url
+"""
+Centralized configuration module for AIQLeads platform.
+All configuration settings should be managed here.
+"""
 
-__all__ = ["get_settings", "get_database_url"]
+from .settings import *
```

Ruff (0.8.2)

`from .settings import *` used; unable to detect undefined names

(F403)